plugins/services/src/js/utils/FrameworkUtil.js

```diff
@@ -1,7 +1,17 @@
+import { Trans } from "@lingui/macro";
+
+import React from "react";
+import { Icon, Legacy } from "@dcos/ui-kit";
```

Comment:
I tried to use the ui-kit tooltip, but not only did it require special styling in the package detail tab, in the framework configuration tab it didn't appear at all.

Reply:
argh! ui-kit's tooltip has `z-index: 2`, which sticks behind `.modal-wrapper`s `z-index: 1000`. cc @mperrotti